app/components/hero/hero.styl

```diff
@@ -116,39 +121,64 @@
             -moz-margin-start 25px
             padding 0 25px
             border-radius 50px
-            background-color #ff00aa
-            background-color var(--active-color)
+            background-color rgba(131, 74, 223, 0.85)
+            border 2px solid var(--white-color)
+            // background-color var(--active-color)
           .sign-in-btn:hover
             // transition color 0.5s linear
             background-color #fff
             background-color var(--white-color)
             color var(--active-color)
           .sign-in-btn:focus
-            background-color #ff00aa
-            background-color var(--active-color)
+            background-color rgba(131, 74, 223, 0.85)
+            // background-color var(--active-color)
             color var(--white-color)
       .header-content
-        padding-top 6%
+        // padding-top 6%
         display flex
-        justify-content center
+        justify-content space-around
         h1
           letter-spacing: 1px
-          font-size 3rem
+          font-size 72px
         .header-content-wrap
           z-index 1
-          width 30%
-          padding 0 2rem
+          width 40%
+          padding 15% 0 0 3%
+          .try-for-free-btn
+            text-transform uppercase
+            font-size: 24px;
+            display: flex;
+            width: 300px;
+            height: 74px;
+            justify-content: center;
+            align-items: center;
+            border-radius 50px
+            border 1px solid transparent
+            /* polyfill for old browsers */
+            background-color #f0a
+            background-color var(--active-color)
+            outline none
+            text-decoration none
+            font-weight 600
+            transition background-color 0.3s linear, border-color 0.3s linear, color 0.3s linear
+            &:hover, &:focus
+              /* polyfill for old browsers */
+              background-color #fff
+              background-color var(--white-color)
+              border-color currentColor
+              color var(--active-color) !important
+            img
+              padding-inline-end 1rem
           .header-content-wrap-text
             a
               padding 15px 40px
               color var(--white-color)
             .header-desc
               padding 20px 0 50px 0
               p
-                font-size 1.2rem
+                font-size 28px
```

Comment:
Why to move from `rem` to `px`? If there is no specific reason, pls set it back to be `rem`